.pre-commit-config.yaml

```diff
@@ -66,4 +66,4 @@ repos:
         args: ["--verbose", "--"]
     -   id: clippy
         # Use 'all-targets' to run on all code, including tests and examples
-        args: ["--all-targets", "--", "-D", "warnings", "-W", "clippy::pedantic"]
+        args: ["--all-targets", "--","-A", "non-snake-case", "-D", "warnings", "-W", "clippy::pedantic"]
```

Comment:
I don't think this is right. We want to keep non-snake-case as a warning and disable only where needed

Reply:
I think it's quite difficult to do this in the macro invocation where one of the warnings was arising. Do you have workarounds for calling it with the derived property macro?